using/product/reputation.md

```diff
@@ -65,21 +65,24 @@ By default, `12` reputation is awarded when a pull request is merged that was op
+| Label           | Reputation    | Examples                                                                                                                                                                                                                                                                                                                                                            |
+| --------------- | ------------- | ------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- |
+| `x:rep/tiny`    | 3 (~5 mins)   | <ul><li>Fixing a single typo or link</li><li>Removing a blank line or adding a line break</li><li>Changing/adding a single code comment</li></ul>                                                                                                                                                                                                                   |
+| `x:rep/small`   | 5 (~10 mins)  | <ul><li>Fixing a single test case, task or example</li><li>Fixing multiple typos or links in a single file</li><li>Clarifying content by adding a few lines to a file</li></ul>                                                                                                                                                                                     |
+| `x:rep/medium`  | 12 (~30 mins) | <ul><li>Syncing an exercise with problem-specifications (incl. edits)</li><li>Adding one or more test cases from scratch</li><li>Improving multiple files in an exercise</li><li>Adding mentor notes for an exercise from scratch</li><li>Fixing a small bug in a test runner/analyzer/representer</li><li>Adding analyzer comments for a single exericse</li></ul> |
+| `x:rep/large`   | 30 (~2 hrs)   | <ul><li>Adding a new concept or practice exercise</li><li>Adding new concept documentation</li><li>Substantial re-writing of an existing concept or exercise</li><li>Adding new CI scripts or other automation</li></ul>                                                                                                                                            |
+| `x:rep/massive` | 100 (~5 hrs)  | <ul><li>Creating a test-runner, analyzer, representer or generator from scratch</li><li>Major refactors to those tools</li><li>Creating major documentation from scratch (e.g. contribution or testing guides)</li></ul>                                                                                                                                            |
```

Comment:
I've slightly rounded the estimated time to make it easier to remember.

Reply:
I don't think the rep + time + examples proposed above match up currently. 
E.g. 
- 30 rep is 120 min -> 4 min per 1 rep, which means tiny would be 1 rep, not 3.
- An example for "large" with 2h is creating a full concept exercise. For anything else than super trivial things like strings, that is not possible in 2h imo, not even for a maintainer. There is a concept to write, a story + exercise to invent etc. Not to mention that theoretically there should be a comprehensive design.md for the exercise and an *exhaustive* about.md for the concept.

Part of this mismatch was already present before the time numbers where added but those make it now more apparent and it might nevertheless be something we want to look into.

I will need some more time to think about what I would propose to fix the issue.

Meanwhile, could you share the base assumption (rep per time unit) you used for the calculation?